src/ctx.rs

```diff
@@ -621,13 +619,7 @@ macro_rules! into_ctx_float_impl {
             #[inline]
             fn into_ctx(self, dst: &mut [u8], le: Endian) {
                 assert!(dst.len() >= $size);
-                write_into!(
-                    signed_to_unsigned!($typ),
-                    $size,
-                    transmute::<$typ, signed_to_unsigned!($typ)>(self),
```

Comment:
just want to be sure this doesn't affect semantics here, the `signed_to_unsigned!($typ)` in the transmute param is ok to replace with just to_bits(), yes?  (I haven't reviewed this code in literally years and years, so my memory for why this stuff is even here is foggy)

Reply:
The [docs](https://doc.rust-lang.org/std/primitive.f32.html#method.to_bits) for this method show, that the unsigned version is always returned.